hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                                    </td>
+                                    <td>
+                                        {{ res.resource_type }}
+                                    </td>
+                                    <td>
+                                        {% if res.deleted_by.first_name %}
+                                            <strong><a href='/user/{{ res.deleted_by.pk }}/' target="_blank">{{res.deleted_by.first_name}} {{res.deleted_by.last_name }}</a></strong>
+                                        {% else %}
+                                            <strong><a href='/user/{{ res.deleted_by.pk }}/' target="_blank">{{ res.deleted_by.username }}</a></strong>
+                                        {% endif %}
+                                    </td>
+                                    <td>
+                                        {{ res.date_deleted }}
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                         {% endif %}
+                        </tbody>
+                    </table>
+                    <div id="clear-deleted-res-warning" style="display: None">It may take seconds to clear deleted resources and update bag. Please keep this window open and wait...</div>
+                </div>
+                <div class="modal-footer">
+                    {% if edit_mode|lower == "true" %}
+                    <button type="button" class="btn btn-success" id="clear-deleted-res-btn" onclick="clear_deleted_resources_table('{{collection_res_id}}');">
+                        Clear and Update Bag
+                    </button>
+                    {% endif %}
+                    <button type="button" class="btn btn-default" data-dismiss="modal" id="save-collection-btn-cancel">
+                       Close
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    <script type="text/javascript" src='{% static "js/jquery.dataTables.js" %}'></script>
+
+    {# put resource specific js below #}
+    <script type="text/javascript">
+
+        var paging = false;
+        var page_length = 15;
+        var page_type = "numbers";
+        var page_length_change = false;
+        $(document).ready(function()
+        {
+            // disable "Publish" btn
+            $("#publish").hide();
+            $("#new-version").hide();
+
+            //------------------ table headers ------------
+            {#                <th>Add</th>             0  #}
+            {#                <th>Title</th>           1  #}
+            {#                <th>Type</th>            2  #}
+            {#                <th>Owners</th>          3  #}
+            {#                <th>Sharing status</th>  4  #}
+            {#                <th>My Permission</th>   5  #}
+            {#                <th>Remove</th>          6  #}
```

Comment:
@zhiyuli OK. Got your point. Just make sure this piece of comments are in sync with the table fields when they are changed in the future then.